### ISSUE TITLE
prop-types rule should not apply to stateless components

### ISSUE THREAD

**Issue by Erid** [yannickcr/eslint-plugin-react#471]
I'm getting that properties are missed for stateless components, but I have no way of defining the propTypes for this kind of components.

I believe this is a false positive.

**Comment by ljharb** [comment]
Absolutely you do - the same way you do with a "class" (using only standardized features):
```js
function Foo(props) {
}
Foo.propTypes = {};

class Bar extends React.Component {
}
Bar.propTypes = {};
```

**Comment by Erid** [comment]
Thanks for the info :+1:.
